src/middlewared/middlewared/plugins/system.py

```diff
@@ -133,6 +155,8 @@ class Config:
 
         `autotune` when enabled executes autotune script which attempts to optimize the system based on the installed
         hardware.
+
+        When `syslogserver` is defined, `sysloglevel` makes sure that logs matching the specified level are sent.
```

Comment:
Not clear what "makes sure" means here.  Sounds like sometimes it doesn't send them.

Reply:
> Not clear what "makes sure" means here. Sounds like sometimes it doesn't send them.

What about:
"When `syslogserver` is defined, the logs matching the `sysloglevel` level are sent." ?

Reply:
`syslog level or above`?

Reply:
@wblock thoughts?

Reply:
That is clearer.  
"When `syslogserver` is defined, logs of `sysloglevel` or above are sent."